### PR TITLE
MINOR:Move 101 card to the upper left

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
Team: I moved the `Kafka 101` card to the upper left corner based on a suggestion from @rspurgeon. Check it out here: 

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/MINOR_move_101_card_to_left/aeaff77af170ad801183c280bea72645a894b0a8/

WDYT?

If everyone agrees, the only other thing to do is hide half of what's on the `Apply a function to data` card and have a `more` link to expose all the tutorials

**Comment by gAmUssA** [comment]
@bbejeck I like it very much! q: shall we have top row like 

`kafka 101 | apply function | aggregate data`

**Comment by ybyzek** [comment]
@bbejeck awesome, love it.

**Comment by bbejeck** [comment]
> shall we have top row like
  kafka 101 | apply function | aggregate data


@gAmUssA yes that would be my preference as well, but we have to get the `more | less` functionality working otherwise the `Kafka 101` card will look silly with 3 tutorials vs 15.

**Comment by bbejeck** [comment]
Updated to put `Apply Function to Data` in the middle and show/hide functionality

View it in staging http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/MINOR_move_101_card_to_left/dbcbfb2998591f1344b341c11b1d795998696a8a/

WDYT?

File: index.html [hunk 1]
```diff
@@ -40,15 +50,16 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
...
+            <li class="more is-hidden"><a href="changing-serialization-format/ksql.html">Convert a stream's serialization format</a></li>
+            <li class="more is-hidden"><a href="split-a-stream-of-events-into-substreams/ksql.html">Split a stream of events into substreams</a></li>
+            <li class="more is-hidden"><a href="merge-many-streams-into-one-stream/ksql.html">Merge many streams into one stream</a></li>
+            <li class="more is-hidden"><a href="finding-distinct-events/ksql.html">Finding distinct events</a></li>
+            <li class="more is-hidden"><a href="udf/ksql.html">Apply a custom, User-Defined Function</a></li>
+            <li class="more is-hidden"><a href="handling-deserialization-errors/ksql.html">Handling deserialization errors</a></li>
+            <li class="more is-hidden"><a href="dynamic-output-topic/kstreams.html">Choosing the output topic dynamically</a></li>
+            <li class="more is-hidden"><a href="naming-stateful-operations/kstreams.html">Naming stateful operations</a></li>
           </ul>
+          <div id="show"><a>More</a></div>
```
Comment (colinhicks) [inline index.html:62]: Very cool. For your consideration, adding some classes to the More anchor would visually differentiate the toggle from the other links ...
```suggestion
          <div id="show"><a class="button is-small is-fullwidth">More</a></div>
```

image

**Comment by bbejeck** [comment]
Thanks for the suggestion @colinhicks, I definitely wanted something more obvious to indicate there's a larger list of tutorials.

**Comment by bbejeck** [comment]
Updated the staged site 

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/MINOR_move_101_card_to_left/86660b32515c75057a61c8637a212eed40417121/

**Comment by gAmUssA** [comment]
Love that "more" button.

**Pull request closed without merging**